lib/matplotlib/patches.py

```diff
@@ -1848,24 +1845,13 @@ def _pprint_styles(_styles):
     _table = [["Class", "Name", "Attrs"]]
 
     for name, cls in sorted(_styles.items()):
-        if six.PY2:
-            args, varargs, varkw, defaults = inspect.getargspec(cls.__init__)
-        else:
-            (args, varargs, varkw, defaults, kwonlyargs, kwonlydefs,
-                annotations) = inspect.getfullargspec(cls.__init__)
-        if defaults:
-            args = [(argname, argdefault)
-                    for argname, argdefault in zip(args[1:], defaults)]
+        spec = inspect.getfullargspec(cls.__init__)
+        if spec.defaults:
+            argstr = ",".join("%s=%s" % pair
+                              for pair in zip(spec.args[-len(spec.defaults):],
+                                              spec.defaults))
```

Comment:
would write this as
```
",".join(map("{}={}".format, spec.args[-len(spec.defaults):], spec.defaults))
```
(the rarely seen but ever useful implicit zip-and-star in multi-arg map).

(would also leave a space after the comma, but that's a separate point)